djangocms_navigation/utils.py

```diff
@@ -0,0 +1,13 @@
+from functools import lru_cache
+
+from django.apps import apps
+
+
+@lru_cache(maxsize=1)
+def supported_models():
+    try:
+        app_config = apps.get_app_config("djangocms_navigation")
```

Comment:
It is generally bad practice to put multiple lines into a try clause unless you have a good reason for it. What can happen is that some line we weren't expecting to throw an exception does and then it gets caught and then someone spends ages debugging (have had this happen on my teams more than once). In this particular case `LookupError` is exotic enough that it's unlikely, but still good to get into good habits. AFAIK, what you probably want here is:
```
try:
    app_config = apps.get_app_config("djangocms_navigation")
except LookupError:
    return []
else:
    extension = app_config.cms_extension
    return extension.navigation_apps_models
```